soc/nordic/common/power.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file Common power.h include for Nordic SoCs.
+ */
+
+#ifndef _ZEPHYR_SOC_ARM_NORDIC_NRF_POWER_H_
+#define _ZEPHYR_SOC_ARM_NORDIC_NRF_POWER_H_
+
+#if defined(NRF_APPLICATION)
+#define RAMBLOCK_CONTROL_BIT_ICACHE MEMCONF_POWER_CONTROL_MEM1_Pos
+#define RAMBLOCK_CONTROL_BIT_DCACHE MEMCONF_POWER_CONTROL_MEM2_Pos
+#define RAMBLOCK_POWER_ID	    0
+#define RAMBLOCK_CONTROL_OFF	    0
+#define RAMBLOCK_RET_MASK	    (MEMCONF_POWER_RET_MEM0_Msk)
+#define RAMBLOCK_RET_BIT_ICACHE	    MEMCONF_POWER_RET_MEM1_Pos
+#define RAMBLOCK_RET_BIT_DCACHE	    MEMCONF_POWER_RET_MEM2_Pos
+#elif defined(NRF_RADIOCORE)
```

Comment:
please, use Zephyr definitions to distinguish between app/radio cores (SOC_...). Also, looks like a 54hx specific header?

Reply:
Moved to nrf54h/soc.h